.openpublishing.redirection.fundamentals.json

```diff
@@ -120,6 +120,206 @@
             "source_path_from_root": "/docs/fundamentals/syslib-diagnostics/syslib1004.md",
             "redirect_url": "/dotnet/fundamentals/syslib-diagnostics/syslib1001"
         },
+        {
```

Comment:
I think you should include 1030 upward. that's where the generator ID's start. if we're missing a 1030-1039 link we should create one.
see
https://github.com/dotnet/runtime/blob/main/docs/project/list-of-diagnostics.md?plain=1#L210

Reply:
also you could anticipate 1090-1118 which are missing but will appear. after all many of these in this PR don't exist yet eg 1046. I suggest just set up redirects for everything from 1030 to 1118

Reply:
> I think you should include 1030 upward. that's where the generator ID's start.

Those IDs have their own pages so no redirects are needed - e.g. https://learn.microsoft.com/en-us/dotnet/fundamentals/syslib-diagnostics/syslib1030.